bcManager/bcManager.py

```diff
@@ -818,14 +855,13 @@ async def update_match_report_from_bc(self, ctx, match):
             report = await self.get_replay_destination(ctx, match)
         
         bapi: ballchasing.Api = self.ballchasing_api[ctx.guild]
-        data = await asyncio.to_thread(
-            bapi.get_replays,
+        data = bapi.get_replays(
```

Comment:
maybe rename to `replays` for consistency and clarity

Reply:
This is the name of the ballchasing library function and has always been. I'm not sure this is necessary.